crates/storage/src/connection/trie.rs

```diff
@@ -290,10 +329,11 @@ impl Transaction<'_> {
                     bincode::config::standard(),
                 )
                 .context("Decoding indices")?;
+                let mut removed = 0;
                 for idx in indices.iter() {
-                    delete_stmt.execute(params![idx]).context("Deleting node")?;
+                    removed += delete_stmt.execute(params![idx]).context("Deleting node")? as u64;
                 }
-                metrics::counter!(METRIC_TRIE_NODES_REMOVED, removed.len() as u64, "table" => table);
+                metrics::counter!(METRIC_TRIE_NODES_REMOVED, removed, "table" => table);
             }
```

Comment:
Can't we use `metrics::counter!(METRIC_TRIE_NODES_REMOVED, indices.len(), "table" => table)` here?